### PR TITLE
LPS-166886

### PR DESCRIPTION
**Pull request by crodriguezy**

### CODE REVIEW

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/expando/expando-test/src/testIntegration/java/com/liferay/expando/service/test/ExpandoValueLocalServiceTest.java [hunk 1]
```diff
@@ -169,6 +170,32 @@ public void testAddNondefaultLocalizedStringValue() throws Exception {
...
+
+		ExpandoValue value = ExpandoTestUtil.addValue(
+			_expandoTable, column,
+			HashMapBuilder.put(
+				_enLocale, "one"
+			).put(
+				_frLocale, "un"
+			).build());
+
+		value = ExpandoValueLocalServiceUtil.getExpandoValue(
```
Comment (dantewang) [inline modules/apps/expando/expando-test/src/testIntegration/java/com/liferay/expando/service/test/ExpandoValueLocalServiceTest.java:191]: @crodriguezy Please avoid direct usage of XLocalServiceUtil in integration tests, we can `@Inject` `ExpandoValueLocalService` directly. Thanks!

**Comment by dantewang** [comment]
@crodriguezy In order to reduce/avoid too many open PRs, I'm going to close this one. Please see my comment above, and resend, thanks!

**Comment by crodriguezy** [comment]
Please see https://github.com/liferay-core-infra/liferay-portal/pull/2461

**Pull request closed without merging**